tests/integration/devfile/cmd_devfile_catalog_test.go

```diff
@@ -79,4 +81,150 @@ var _ = Describe("odo devfile catalog command tests", func() {
 			helper.MatchAllInOutput(output, []string{"name: nodejs-starter", "Registry: " + registryName})
 		})
 	})
+	Context("When executing catalog describe component with a component name that does not have a devfile component", func() {
+		It("should print message that there is no Odo devfile component available", func() {
+			output := helper.CmdShouldPass("odo", "catalog", "describe", "component", "java")
+			helper.MatchAllInOutput(output, []string{"There are no Odo devfile components with the name \"java\""})
+		})
+	})
+	Context("When executing catalog describe component with more than one argument", func() {
```

Comment:
Could you please remove all these test spec https://github.com/openshift/odo/pull/3935/files#diff-318517042700080474192fc7dc04e456R90-R101 from integration test and add these scenario in the issue https://github.com/openshift/odo/issues/4052